integration-test/src/test/java/org/sagebionetworks/IT070SynapseJavaClientTrashCanTest.java

```diff
@@ -201,4 +201,39 @@ public void testAdmin() throws SynapseException {
 		child = null;
 		parent = null;
 	}
+	
+	@Test
+	public void testAdminPurgeLeaves() throws SynapseException{
+		final long limit = Long.MAX_VALUE;
+		final long offset = 0;
+		final long daysInTrash = 0;
+		
+		//reset trash can
+		adminSynapse.purgeTrash();
+		
+		//move the 2 nodes to the trash
+		synapse.moveToTrash(parent.getId());
+		PaginatedResults<TrashedEntity> results = adminSynapse.viewTrash(0L, Long.MAX_VALUE);
+		assertNotNull(results);
+		assertEquals(2, results.getResults().size());
+		assertEquals(2, results.getTotalNumberOfResults());
+		
+		//purge the trash leaves (child node)
+		adminSynapse.purgeTrashLeaves(daysInTrash, limit);
+		
+		//check parent still in trash
+		results = adminSynapse.viewTrash(offset, limit);
+		assertNotNull(results);
+		assertEquals(1, results.getResults().size());//only 1 item in trash
+		assertEquals(parent.getId(), results.getResults().get(0).getEntityId());//id of trash item matches parent node's id
+		
+		//purge leaves again to empty trash can
+		adminSynapse.purgeTrashLeaves(daysInTrash, limit);
+		results = adminSynapse.viewTrash(offset, limit);
+		assertNotNull(results);
+		assertEquals(0, results.getResults().size());//nothing in trash
+		
+		child = null;
```

Comment:
use try catch in after to clean up in case other dev forget to set the value to null